src/main/java/org/wildfly/plugins/bombuilder/BuildBomMojo.java

```diff
@@ -225,6 +233,10 @@ public enum InheritExclusions {
     private final PomDependencyVersionsTransformer versionsTransformer;
     private final ModelWriter modelWriter;
 
+
+    @Parameter(alias = "channels", required = false)
+    private List<ChannelConfiguration> channels;
```

Comment:
@emmartins `ChannelConfiguration` class is missing. I don't think it's one from channel library...